homework3_mysql/homework3.sql

```diff
@@ -0,0 +1,391 @@
+"Сучасна проза",
+9786171270909,
+240,
+"Тосікадзу",
+"Кавагуті",
+"kavaguti@gmail.com",
+68,
+"1951-10-26",
+78
+),
+
+("Містер Мерседес",
+"Спогади про нерозкритий злочин не дають спокою колишньому поліцейському Біллу Годжесу. Він мав упіймати злочинця, який на вкраденому «мерседесі» навмисне вбив та покалічив десятки людей, але…",
+"Сучасна проза",
+9786171274754,
+544,
+"Стівен",
+"Кінг",
+"king@gmail.com",
+72,
+"1947-09-21",
+252
+);
+
+insert into book(book_name,publication_date,category,isbn,number_of_pages,author_name,autor_surname,autor_email,autor_age,autor_birthday,price)
+values
+("Ліки від кохання",
+"1985-07-16",
+"Зарубіжна література",
+9786171222700,
+415,
+"Ірвін",
+"Ялом",
+"irvin@gmail.com",
+68,
+"1951-09-21",
+110
+),
+
+("Нова стара баба",
+"2013-01-01",
+"Роман",
+9789661463041,
+268,
+"Лариса",
+"Денисенко",
+"denysenlo@gmail.com",
+46,
+"1973-06-17",
+125
+),
+
+("Тому,що ти є",
+"2012-03-02",
+"Роман",
+9789661423557,
+236,
+"Дара",
+"Корній",
+"korniy@gmail.com",
+49,
+"1970-09-20",
+115
+),
+
+("Аномалія",
+"2012-04-02",
+"Роман",
+9786171274358,
+384,
+"Андрій",
+"Новік",
+"novik@gmail.com",
+49,
+"1970-08-22",
+243
+),
+
+("Характерник",
+"2016-05-10",
+"Історико-пригодницький роман",
+9786171268418,
+304,
+"Василь",
+"Шкляр",
+"shkliar@gmail.com",
+68,
+"1951-06-10",
+223
+),
+
+("Букова земля",
+"2016-04-05",
+"Роман",
+9786175851791,
+928,
+"Марія",
+"Матіос",
+"matios@gmail.com",
+70,
+"1950-08-16",
+213
+),
+
+("Бот",
+"2017-06-05",
+"Сучасна проза",
+9786171268340,
+544,
+"Макс",
+"Кідрук",
+"kidruk@gmail.com",
+35,
+"1984-04-01",
+300
+),
+
+("Втрачений символ",
+"2008-02-02",
+"Роман",
+9786171249554,
+608,
+"Ден",
+"Браун",
+"braun@gmail.com",
+55,
+"1964-06-22",
+305
+),
+
+("Квіти для Елджернона",
+"2004-01-28",
+"Роман",
+9789661476522,
+304,
+"Деніел",
+"Кіз",
+"kiz@gmail.com",
+70,
+"1950-08-16",
+203
+),
+
+("Подаруй мені зірку",
+"2014-01-27",
+"Роман",
+9786171270961,
+400,
+"Джоджо",
+"Мойєс",
+"moes@gmail.com",
+50,
+"1969-08-04",
+152
+),
+
+("Метелик",
+"2010-03-12",
+"Роман",
+9786171258068,
+576,
+"Анрі",
+"Шар'єр",
+"anri@gmail.com",
+66,
+"1954-09-22",
+99
+);
+
+insert into book(book_name,category,isbn,number_of_pages,author_name,autor_surname,autor_email,autor_age,autor_birthday,autor_awards,price)
+values
+("Агнес грей",
+"Зарубіжна література",
+9786171276123,
+288,
+"Енн",
+"Бронте",
+"bronte@gmail.com",
+29,
+"1820-01-17",
+"Коронація слова",
+98
+),
+
+("Бог завжди подорожує інкогніто",
+"Роман",
+9786171215528,
+416,
+"Лоран",
+"Гунель",
+"gunel@gmail.com",
+53,
+"1966-08-10",
+"Гранослов",
+104.3456
+),
+
+("Жовта книга",
+"Роман",
+9789661408073,
+142,
+"Ірена",
+"Карпа",
+"karpa@gmail.com",
+29,
+"1980-12-08",
+"Коронація слова",
+87.5429
+),
+
+("Психологія впливу",
+"Сучасна проза",
+9786171233522,
+352,
+"Роберт",
+"Чалдині",
+"chaldyni@gmail.com",
+53,
+"1966-09-10",
+"Коронація слова",
+92.52
+);
+
+
+select * from book;
+set sql_safe_updates = 0;
```

Comment:
Only a few statements require sql_safe_updates = 0, so you could move this line to only execute this before those lines.